app/controllers/psychologist/registrations_controller.rb

```diff
@@ -7,13 +7,26 @@ class Psychologist::RegistrationsController < Devise::RegistrationsController
   def new
     super
   end
+   
+  def update
+    super
+  end
+
+  protected
+
+  def after_update_path_for(profile_psychologist)
+    puts 'this is happening yoyo mama'
```

Comment:
@xFerchoVJ  Do you really need this `puts` in your code?

Reply:
Nooo
I forgot to delete that, sorry!